l3kernel/CHANGELOG.md

```diff
@@ -15,9 +15,12 @@ this project uses date-based 'snapshot' version identifiers.
 - Several missing `\tex_<name>:D` primitives from LuaTeX and (up)TeX
 
 ### Changed
+
```

Comment:
It seems by convention this empty line is surplus.

Reply:
sorry I just missed that before merge, but the changelog will get further edits anyway before it goes out. Thanks for your review